core/src/com/itemshop/factories/tiles/GrassFactory.java

```diff
@@ -15,17 +18,22 @@
 	/**
 	 * Creates the entity.
 	 * @param engine The engine to add the entity to.
+	 * @param random The rng to use.
 	 * @param x The x position.
 	 * @param y The y position.
 	 */
-	public void create(Engine engine, int x, int y) {
+	public void create(Engine engine, Random random, int x, int y) {
```

Comment:
Potentially could've passed this in via the constructor to avoid needing to add it to all the create() methods